src/components/visual/explorer-link/explorer-link.tsx

```diff
@@ -26,12 +26,23 @@ export class ExplorerLink {
     let icon = 'faArrowUpRightFromSquare';
 
     if (this.icon) {
-      icon = typeof this.icon === 'string' ? getIconHtmlFromIconName(this.icon) : getIconHtmlFromIconDefinition(this.icon);
+      icon =
+        typeof this.icon === 'string' ? getIconHtmlFromIconName(this.icon) : getIconHtmlFromIconDefinition(this.icon);
     }
 
     return (
-      <a data-testid={this.dataTestId} href={this.link} target="_blank" class={{ 'mvx:explorer-link': true, [this.class]: Boolean(this.class) }} rel="noreferrer">
-        {this.hasSlotContent ? <slot name="content"></slot> : this.text ?? <mvx-fa-icon icon={icon} class={this.iconClass}></mvx-fa-icon>}
+      <a
+        data-testid={this.dataTestId}
+        href={this.link}
+        target="_blank"
+        class={{ 'mvx:explorer-link': true, [this.class]: Boolean(this.class) }}
+        rel="noreferrer"
+      >
+        {this.hasSlotContent ? (
+          <slot name="content"></slot>
+        ) : (
+          this.text ?? <mvx-fa-icon icon={icon} class={this.iconClass}></mvx-fa-icon>
```

Comment:
I find hasSlotContent & text to be overkill
why not use slot name="content"
and slot name="text" ?